src/js/jsx/sections/libraries/assets/AssetPreviewImage.jsx

```diff
@@ -43,30 +43,60 @@ define(function (require, exports, module) {
         
         getInitialState: function () {
             return {
-                loading: true,
-                renditionPath: null
+                loading: false,
+                renditionPath: null,
+                lastModified: null
```

Comment:
I moved `lastModified` out of the state as it is not related to component updates, also made the code more clear by moving the `lastModified` assignment to the `updateRenditionPath` function.